crates/wasmtime/src/module/registry.rs

```diff
@@ -33,24 +33,6 @@ fn func_by_pc(module: &CompiledModule, pc: usize) -> Option<(DefinedFuncIndex, u
 pub struct ModuleRegistry(BTreeMap<usize, Arc<RegisteredModule>>);
 
 impl ModuleRegistry {
-    /// Fetches frame information about a program counter in a backtrace.
-    ///
-    /// Returns an object if this `pc` is known to some previously registered
-    /// module, or returns `None` if no information can be found. The boolean
-    /// returned indicates whether the original module has unparsed debug
-    /// information due to the compiler's configuration.
-    pub fn lookup_frame_info(&self, pc: usize) -> Option<(FrameInfo, bool)> {
```

Comment:
The movement here was basically just movement of code from the `ModuleRegistry` to the `GlobalModuleRegistry`. These functions should only be accessed during trap creation so the slow locking should be ok, and it means that we didn't need the store during trap creation which was made a few things easier. 

Data-representation-wise, though, nothing changed. The global registry already had all the information we needed.